src/particles/ImpactXParticleContainer.cpp

```diff
@@ -124,7 +125,19 @@ namespace impactx
         // number of particles to add
         int const np = x.size();
 
-        auto& particle_tile = DefineAndReturnParticleTile(0, 0, 0);
+        // we add particles to lev 0, tile 0 of the first box assigned to this proc
+        int lid = 0, gid = 0, tid = 0;
+        {
+            const auto& pmap = ParticleDistributionMap(lid).ProcessorMap();
+            auto it = std::find(pmap.begin(), pmap.end(), amrex::ParallelDescriptor::MyProc());
+            if (it == std::end(pmap)) {
+                amrex::Abort("Attempting to add particles to box that does not exist.");
+            } else {
+                gid = *it;
+            }
+        }
+        auto& particle_tile = DefineAndReturnParticleTile(lid, gid, tid);
```

Comment:
@atmyers @WeiqunZhang for `ParIter` loops after initialization, with OpenMP parallelism, does it matter which `tid` we use?

Or will be have now always one tile and no on-rank parallelism?

Reply:
(Please note we never call `pc.Redistribute()` in this mode)